pensjon-brevbaker/build.gradle.kts

```diff
@@ -87,7 +87,7 @@ dependencies {
     implementation("io.ktor:ktor-server-netty:$ktorVersion")
     implementation("io.ktor:ktor-server-status-pages:$ktorVersion")
     implementation("net.logstash.logback:logstash-logback-encoder:$logstashVersion")
-    implementation("no.nav.pensjon.brev:pensjon-brevbaker-api-model:3.5.4")
+    implementation("no.nav.pensjon.brev:pensjon-brevbaker-api-model:3.5.5-SNAPSHOT")
```

Comment:
make it 3.5.5 not snapshot. Sapshot should only be used while working locally.

Reply:
Updated build gradle